modules/support-generating-a-sosreport-archive.adoc

```diff
@@ -85,6 +85,13 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ----
```

Comment:
`sosreport` is deprecated already with latest toolbox container. 
so i recommend to replace `sosreport` with `sos report`. we still need to check if those -k options still work with sos report command.

Reply:
@pperiyasamy are you saying that we should be using `sos report` and not `sosreport` in this document?

Reply:
yes, correct. it must be `sos report` and remaining arguments are the same.